src/CommunityVoices/App/Api/Component/SecureContainer.php

```diff
@@ -24,15 +24,17 @@ public function contain($decoratedInstance)
             $user = $this->identifier->identify();
 
             if (!is_object($contained)) {
+                // This should be more verbose, but we cannot print out a class name
+                // of a non-object. So, what to do here?
                 throw new \Exception('No decorated instance!');
             }
 
+            $signature = get_class($contained) . "::" . $method;
+
             if (!method_exists($contained, $method)) {
-                throw new \Exception('Method not found');
+                throw new \Exception('Method not found ' . $signature);
```

Comment:
Method '%signature%' not found I like better

Reply:
Hmm... I'm really used to error messages being in the format of the error message followed by the description of the specific case that helps. (e.g. Array index out of bounds: 4) Are you sure you prefer this?  It makes it harder to find what signature is causing the problem at a glance.